aspnetcore/fundamentals/middleware/index.md

```diff
@@ -209,6 +227,7 @@ ASP.NET Core ships with the following middleware components. The *Order* column
 | Middleware | Description | Order |
 | ---------- | ----------- | ----- |
 | [Authentication](xref:security/authentication/identity) | Provides authentication support. | Before `HttpContext.User` is needed. Terminal for OAuth callbacks. |
+| [Cookie Policy](xref:security/gdpr) | Ask for (and track) consent from users for storing personal information. | Before Session Middleware, if present. |
```

Comment:
It does not request consent, it only tracks it. It also enforces minimum standards for cookie fields like `secure` and `SameSite`.

Reply:
@Tratcher There's a consent button, and this remark on consent in the topic ...

> Obtains the state of tracking for the user. If the app is configured to require consent, the user must consent before cookies can be tracked.

I shamelessly stole the line in the table almost word-for-word from the topic ... see the 2nd bullet of the introduction at :point_right: https://docs.microsoft.com/aspnet/core/security/gdpr

Do I need to make changes to both topics in addition to changing this remark?

Reply:
The template UI that shows a consent button is independent of the middleware. Consent is requested and granted through the UI, and then the middleware tracks that result for future requests. The GDPR doc is fine because it's talking about the end-to-end scenario.

Reply:
Gotcha ... thanks.